### PR TITLE
Xw 267

### PR DESCRIPTION
**Pull request by justnpT**
Adding tests for Weibo widget on oasis and mercury

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/widget/WeiboWidgetPageObject.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+  @FindBy(css = "iframe[data-wikia-widget='weibo']")
+  private WebElement weiboIframe;
+  @FindBy(css = "div.tsina_open")
+  private WebElement weiboBody;
+
+  private static final String TAG_NAME = "weibo";
+  private static final String ARTICLE_NAME = "WeiboWidget";
+  private static final String TAG =
+      "<weibo uids=\"1642909335,1782515283\">";
```
Comment (bognix) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/widget/WeiboWidgetPageObject.java]: close the tag so it's correct

File: src/test/java/com/wikia/webdriver/testcases/widgettests/WeiboTests.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+import org.testng.annotations.Test;
+
+/**
+ * @ownership: Content X-Wing
+ */
+public class WeiboTests extends NewTestTemplate {
+
+  @Test
+  @Execute(onWikia = "mercuryautomationtesting")
```
Comment (bognix) [inline src/test/java/com/wikia/webdriver/testcases/widgettests/WeiboTests.java]: test for oasis should also run on mercuryautomationtesting wiki?

Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/testcases/widgettests/WeiboTests.java]: It doesn't matter, there won't be any collision

File: src/test/java/com/wikia/webdriver/testcases/widgettests/TwitterTests.java [hunk 3]
```diff
@@ -15,12 +15,12 @@
 
   @Test
   @Execute(onWikia = "mercuryautomationtesting")
-  public void TwitterWidgetTest_001_isLoaded() {
-    TwitterWidgetPageObject twitterWidgetPageObject = new TwitterWidgetPageObject(driver);
+  public void WidgetTest_001_Twitter_isLoaded() {
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/testcases/widgettests/TwitterTests.java]: Previous name was better

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/widgettests/WeiboTests.java [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    weiboWidget.createAndNavigate(wikiURL);
+
+    new NavigationSideComponentObject(driver)
+        .navigateToArticle(MAPS_ARTICLE_NAME)
+        .navigateToArticle(WEIBO_ARTICLE_NAME);
+
+    Assertion
+        .assertTrue(weiboWidget.isLoadedOnMercury(), MercuryMessages.INVISIBLE_MSG);
+  }
+
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/widgettests/WeiboTests.java]: Redundant line

**Comment by tomasznapieralski** [comment]
LGTM

**Pull request merged**